src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,26 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: repeat(1, $card-width);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, $card-width);
```

Comment:
Ensure that the variable `$card-width` is defined in your SCSS files. If not, define it to avoid compilation errors.